scripts/ci/batch_lhci_leak.sh

```diff
@@ -0,0 +1,141 @@
+#!/bin/bash
+set -e
+NETWORK_NAME=${NETWORK_NAME:-"website-network"}
+WEBSITE_DOMAIN=${WEBSITE_DOMAIN:-"localhost"}
+DEV_PORT=${DEV_PORT:-"3000"}
+NEXT_PUBLIC_PROD_PORT=${NEXT_PUBLIC_PROD_PORT:-"3001"}
+PLAYWRIGHT_TEST_PORT=${PLAYWRIGHT_TEST_PORT:-"9323"}
+UI_HOST=${UI_HOST:-"0.0.0.0"}
+PROD_CONTAINER_NAME=${PROD_CONTAINER_NAME:-"website-prod"}
+DOCKER_COMPOSE_DEV_FILE=${DOCKER_COMPOSE_DEV_FILE:-"docker-compose.yml"}
+DOCKER_COMPOSE_TEST_FILE=${DOCKER_COMPOSE_TEST_FILE:-"docker-compose.test.yml"}
+COMMON_HEALTHCHECKS_FILE=${COMMON_HEALTHCHECKS_FILE:-"common-healthchecks.yml"}
+echo "🐳 DIND Environment Setup Script"
+echo "================================"
+setup_docker_network() {
+    echo "📡 Setting up Docker network..."
+    docker network create "$NETWORK_NAME" 2>/dev/null || echo "Network $NETWORK_NAME already exists"
+    echo "✅ Docker network configured"
```

Comment:
Please remove echo after each command inside all bash scripts in scripts/ci directories 
That is hard to read and maintain